services/site/src/pages/legal-notice.tsx

```diff
@@ -36,40 +36,38 @@ const Imprint: React.FunctionComponent = () => {
       <Navigation />
       <main className={clsx("h-full box-border max-w-screen-3xl mx-auto")}>
         <div className={clsx("mx-8 py-8 h-main max-w-screen-3xl mt-12", "sm:mx-12 sm:mt-24", "lg:mx-24")}>
-          <h1 className={clsx("mb-8", "h2", "md:h1")}>Imprint</h1>
+          <h1 className={clsx("mb-8", "h2", "md:h1")}>Legal Notice</h1>
           <p className={clsx("italic mb-8")}>
             Information obligation according to §5 E-Commerce Law, §14 Corporate Code, §63 Trade Regulations and disclosure obligation according to
             §25 Media Act.
           </p>
 
-          <p>
-            <span aria-label="Allyphant">a11yphant</span> is a master's project by six students at the Salzburg University of Applied Sciences. We
-            tried our best to make this website accessible. However, we are no experts yet, so please let us know if we missed something.
-          </p>
-
+          <h2 className={clsx("h5", "sm:h4")}>Contact Information</h2>
           <address className={clsx("not-italic mb-12")}>
-            If you are interested in working with us or just want to know more about the project, simply{" "}
-            <Link href={"mailto:info@a11yphant.com"}>
-              <a
-                className={clsx(
-                  "text-light font-sans font-normal border-light",
-                  "transition-colors duration-300",
-                  "hover:text-primary-light hover:border-transparent",
-                  "focus-rounded-instead-of-underline",
-                )}
-              >
-                drop us a line
-              </a>
-            </Link>
-            . <br /> <br />
-            <strong>University of Applied Sciences Salzburg</strong>
-            <br />
-            Urstein Süd 3 <br />
-            5412 Puch bei Hallein, Austria
+            <p>
```

Comment:
Add my name here before the break. so people know whom to address. Then it is equal to the Privacy Page:

<strong>Daniela Kubesch, BSc.</strong>

Or, if you don't like that, please remove my name from the privacy policy. It just needs to be the same on both pages.